frontend/lib/src/components/elements/Tabs/Tabs.tsx

```diff
@@ -174,6 +176,7 @@ function Tabs(props: TabProps): ReactElement {
 
           return (
             <UITab
+              data-testid={"stTab"}
```

Comment:
nit: I think you can leave out the `{}` brackets around the string

Reply:
I changed it 👍